TTHAnalysis/python/plotter/run2.sh

```diff
@@ -0,0 +1,40 @@
+python mcPlots.py RunII_SM_WZ/2016/mca_wz.txt RunII_SM_WZ/2016/cuts_wzsm.txt RunII_SM_WZ/2016/plots_wz.txt --tree treeProducerSusyMultilepton -P /pool/ciencias/HeppyTrees/RA7/nanoAODv4_2016_skimWZ/ {P}/leptonPtCorrections/ --FMCs {P}/trigger_prefiring/ --Fs {P}/leptonJetReCleanerWZSM/ --Fs {P}/leptonBuilderWZSM_v2/ --Fs {P}/trigger_2016/ --FMCs {P}/leptonMatcher/ --FMCs {P}/bTagWeights/ -L RunII_SM_WZ/functionsSF.cc -L RunII_SM_WZ/functionsMCMatch.cc --mcc RunII_SM_WZ/2016/mcc_triggerdefs.txt --maxRatioRange 0. 3.0 --fixRatioRange --print C,pdf,png,txt --legendWidth 0.23 --legendFontSize 0.036 --showMCError --showRatio --perBin -W 'getLeptonSF_v4(0,0,2016,LepGood_pt[0],LepGood_eta[0],LepGood_pdgId[0])*getLeptonSF_v4(0,0,2016,LepGood_pt[1],LepGood_eta[1],LepGood_pdgId[1])*getLeptonSF_v4(0,0,2016,LepGood_pt[2],LepGood_eta[2],LepGood_pdgId[2])*puWeight*bTagWeightDeepCSVT*weight_PrefiringJets*weight_PrefiringPhotons' --obj Events -j 6 -l 35.9 -f --pdir /nfs/fanae/user/joanrs/www/2016/wz --showRatio -E SRWZ --sP mtW3l --sP flavor3l --sP charge3l
```

Comment:
Better not to commit all your private "save this specific code line" scripts, we do have a centralized bookeeping one in: RunII_SM_WZ/utils/doAnal.py that you are more than welcome to update.